### PR TITLE
[1/2]docs copyedit for the ansible.utils collection

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Also ran the collection_prep_add_docs so it's picking up some new docs rst changes.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

ansible.utils
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by samccann** [comment]
...ignore the [1/2] - that's a sprint tracking thing we are attempting in docs land.

**Review by ganeshrn — APPROVED** [review]
Thank you!

File: docs/ansible.utils.get_path_filter.rst [hunk 1]
```diff
@@ -70,8 +70,8 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The variable from which the value should be extraced</div>
-                        <div>This option represents the value that is passed to filter plugin in pipe format.</div>
+                        <div>The variable from which the value should be extraced.</div>
```
Comment (acozine) [inline docs/ansible.utils.get_path_filter.rst:73]:
```suggestion
                        <div>The variable from which the value should be extracted.</div>
```

File: docs/ansible.utils.get_path_lookup.rst [hunk 2]
```diff
@@ -69,7 +69,7 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The variable from which the value should be extraced</div>
+                        <div>The variable from which the value should be extraced.</div>
```
Comment (acozine) [inline docs/ansible.utils.get_path_lookup.rst:72]:
```suggestion
                        <div>The variable from which the value should be extracted.</div>
```

File: docs/ansible.utils.to_paths_filter.rst [hunk 3]
```diff
@@ -18,9 +18,9 @@ Version added: 1.0
 Synopsis
 --------
 - Flatten a complex object into a dictionary of paths and values.
-- Paths are dot delimited whenever possible
-- Brakets are used for list indicies and keys that contain special characters
-- ``to_paths`` is also available as a ``lookup plugin`` for convenience
+- Paths are dot delimited whenever possible.
+- Brakets are used for list indicies and keys that contain special characters.
```
Comment (acozine) [inline docs/ansible.utils.to_paths_filter.rst:22]:
```suggestion
- Brakets are used for list indices and keys that contain special characters.
```

File: docs/ansible.utils.to_paths_filter.rst [hunk 4]
```diff
@@ -52,7 +52,7 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>Prepend each path entry. Useful to add the initial <code>var</code> name.</div>
+                        <div>Prepend each path entry. Useful to add the initial <code>var</code>. name.</div>
```
Comment (acozine) [inline docs/ansible.utils.to_paths_filter.rst:55]: This looks odd - should there be another sentence at the end? One that ends with `name`?

File: docs/ansible.utils.to_paths_lookup.rst [hunk 5]
```diff
@@ -52,7 +52,7 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>Prepend each path entry. Useful to add the initial <code>var</code> name.</div>
+                        <div>Prepend each path entry. Useful to add the initial <code>var</code>. name.</div>
```
Comment (acozine) [inline docs/ansible.utils.to_paths_lookup.rst:55]: Same thing here. Or was this meant to be `var.name`? Or just `var` and delete "name"?

File: docs/ansible.utils.update_fact_module.rst [hunk 6]
```diff
@@ -19,7 +19,7 @@ Synopsis
 --------
 - This module allows updating existing variables.
 - Variables are updated on a host-by-host basis.
-- Variable are not modified in place, instead they are returned by the module
+- Variable are not modified in place, instead they are returned by the module.
```
Comment (acozine) [inline docs/ansible.utils.update_fact_module.rst:22]:
```suggestion
- Variables are not modified in place, instead they are returned by the module.
```

File: docs/ansible.utils.validate_filter.rst [hunk 7]
```diff
@@ -49,9 +49,9 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The criteria used for validation of value that represents <code>data</code> options.</div>
-                        <div>This option represents the first argument passed in the filter plugin For example <em>config_data|ansible.utils.validate(config_criteria</em>), in this case the value of <em>config_criteria</em> represents this option.</div>
-                        <div>For the type of <code>criteria</code> that represents this value refer documentation of individual validator plugins.</div>
+                        <div>The criteria used for validation of value that represents <code>data</code>. options.</div>
```
Comment (acozine) [inline docs/ansible.utils.validate_filter.rst:52]: Same question here - it looks like a stray `.` or a search/replace error, maybe?

File: docs/ansible.utils.validate_lookup.rst [hunk 8]
```diff
@@ -49,9 +49,9 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The criteria used for validation of value that represents <code>data</code> options.</div>
+                        <div>The criteria used for validation of value that represents <code>data</code>. options.</div>
```
Comment (acozine) [inline docs/ansible.utils.validate_lookup.rst:52]: Same here.

File: plugins/filter/get_path.py [hunk 9]
```diff
@@ -14,29 +14,29 @@
...
-        - C(get_path) is also available as a C(lookup plugin) for convenience
+        - Use a C(path) to retrieve a nested value from a C(var).
+        - C(get_path) is also available as a C(lookup plugin) for convenience.
         - Using the parameters below- C(var|ansible.utils.get_path(path, wantlist))
     options:
       var:
         description:
-        - The variable from which the value should be extraced
-        - This option represents the value that is passed to filter plugin in pipe format.
+        - The variable from which the value should be extraced.
```
Comment (acozine) [inline plugins/filter/get_path.py:26]:
```suggestion
        - The variable from which the value should be extracted.
```

File: plugins/filter/to_paths.py [hunk 10]
```diff
@@ -15,24 +15,24 @@
...
         description:
         - The value of C(var) will be will be used.
-        - This option represents the value that is passed to filter plugin in pipe format.
+        - This option represents the value that is passed to the filter plugin in pipe format.
         - For example I(config_data|ansible.utils.to_paths()), in this case I(config_data) represents this option.
         type: raw
         required: True
       prepend:
-        description: Prepend each path entry. Useful to add the initial C(var) name.
+        description: Prepend each path entry. Useful to add the initial C(var). name.
```
Comment (acozine) [inline plugins/filter/to_paths.py:35]: Here too.

File: plugins/filter/validate.py [hunk 11]
```diff
@@ -13,36 +13,36 @@
...
           represents this option.
-        - For the type of C(data) that represents this value refer documentation of individual validator plugins.
+        - For the type of C(data) that represents this value refer to the documentation of individual validator plugins.
         required: True
       criteria:
         type: raw
         description:
-        - The criteria used for validation of value that represents C(data) options.
-        - This option represents the first argument passed in the filter plugin
+        - The criteria used for validation of value that represents C(data). options.
```
Comment (acozine) [inline plugins/filter/validate.py:25]: And here.

File: plugins/lookup/get_path.py [hunk 12]
```diff
@@ -15,28 +15,28 @@
...
     short_description: Retrieve the value in a variable using a path
     description:
-        - Use a C(path) to retreive a nested value from a C(var)
+        - Use a C(path) to retrieve a nested value from a C(var)
         - C(get_path) is also available as a C(filter plugin) for convenience
         - Using the parameters below- C(lookup('ansible.utils.get_path', var, path, wantlist))
     options:
       var:
-        description: The variable from which the value should be extraced
+        description: The variable from which the value should be extraced.
```
Comment (acozine) [inline plugins/lookup/get_path.py:26]:
```suggestion
        description: The variable from which the value should be extracted.
```

File: plugins/lookup/to_paths.py [hunk 13]
```diff
@@ -15,21 +15,21 @@
...
+        - C(to_paths) is also available as a filter plugin.
         - Using the parameters below- C(lookup('ansible.utils.to_paths', var, prepend, wantlist))
     options:
       var:
         description: The value of C(var) will be will be used.
         type: raw
         required: True
       prepend:
-        description: Prepend each path entry. Useful to add the initial C(var) name.
+        description: Prepend each path entry. Useful to add the initial C(var). name.
```
Comment (acozine) [inline plugins/lookup/to_paths.py:32]: Here too.

File: plugins/lookup/validate.py [hunk 14]
```diff
@@ -18,37 +18,37 @@
...
           For example I(lookup(config_data, config_criteria, engine='ansible.utils.jsonschema')),
           in this case I(config_data) represents this option.
-        - For the type of C(data) that represents this value refer documentation of individual validate plugins.
+        - For the type of C(data) that represents this value refer to the documentation of individual validate plugins.
         required: True
       criteria:
         type: raw
         description:
-        - The criteria used for validation of value that represents C(data) options.
+        - The criteria used for validation of value that represents C(data). options.
```
Comment (acozine) [inline plugins/lookup/validate.py:30]: Again.

File: plugins/modules/update_fact.py [hunk 15]
```diff
@@ -17,26 +17,26 @@
 description:
     - This module allows updating existing variables.
     - Variables are updated on a host-by-host basis.
-    - Variable are not modified in place, instead they are returned by the module
+    - Variable are not modified in place, instead they are returned by the module.
```
Comment (acozine) [inline plugins/modules/update_fact.py:20]:
```suggestion
    - Variables are not modified in place, instead they are returned by the module.
```

**Review by acozine — COMMENTED** [review]
Great work on a very useful set of documentation pages. I found a few nits to pick and had a question about the lines that end with `.` and then a single word and another `.`.

File: docs/ansible.utils.get_path_filter.rst (hunk 1 above)
Comment (samccann) [reply docs/ansible.utils.get_path_filter.rst:73]: These files cannot be edited directly here in github so I'll resolve the conversations as I fix them locally.

**Comment by samccann** [comment]
Implemented all of acozine's feedback.. Thanks!

**Pull request merged**